components/NavigationBar.js

```diff
@@ -22,8 +24,37 @@ export default function NavigationBar () {
 function NavigationLink ({ href, disabled, children }) {
   return (
     <span className={disabled ? 'disabled' : ''}>
-      <LevitatingLink href={href}>{children}</LevitatingLink>
-      <style jsx>{`
+      <Link to={href} spy smooth duration={500}>
+          <div className='nav-link'>{children}</div>
+      </Link>
+      <style jsx>{` 
+        .nav-link{
+          color: #fff;
+          text-decoration: none;
+          display: inline-block;
+          position: relative;
+          transition: 0.2s transform;
+          transform: translateY(0);
+        }
+        .nav-link:hover{
+          cursor: pointer;
+          transform: translateY(-2px);
+        }
+        .nav-link::after {
```

Comment:
This code is copied from the `LevitatingLink` component. Having to copy-and-paste code like this isn’t probably the correct way to solve this problem 😉.